community/pmc-tool/collector/main.py

```diff
@@ -59,25 +59,18 @@
 create_pull_requests_table(conn)
 
 # Retrieve list of pull requests
-authors_count = {}
-
+pull_requests = query_prs(API_ENDPOINT, headers)
```

Comment:
the `API_ENDPOINT` never changes when we load PRs, right? can you please move the API_ENDPOINT variable to the `gh.py` file? it belongs there.

The query_prs can reeive the `owner` and the `repo` parameters, but `headers` and `API_ENDPOINT` will never change, right? so it should live under `gh.py`.